scripts/build.ps1

```diff
@@ -55,6 +55,10 @@ Param(
 
 # Dot source util functions into this scope
 . ".\utils.ps1"
+
+# We need these for add-cmderplugin.ps1
+copy ".\utils.ps1" ../bin
```

Comment:
Is it really needed to copy a utils files into the bin directory (which is as I understand it for user scripts) and not used it via  `CMDER_ROOT`?